src/main/java/frc/team166/robot/Robot.java

```diff
@@ -30,12 +31,12 @@
     //this just adds a drive subsystem
     public static final Drive drive = new Drive();
     public static final Manipulator manipulator = new Manipulator();
+    public static final LED led = new LED();
```

Comment:
This needs to exist before manipulator so that commands that use the LED class will work correctly.
It should probably be the first subsystem we initialize.